changelog.d/pa-2958.added

```diff
@@ -0,0 +1,22 @@
+taint-mode: Added **experimental** `control: true` option to `pattern-sources`,
+e.g.:
+
+```yaml
+    pattern-sources:
+      - control: true
+        pattern: source(...)
+```
+
+Such sources taint the "control flow" (or the program counter) so that it is
```

Comment:
Nice, but who asked for this? SR?

Reply:
Three or four customers!

Reply:
And I heard that SSC could find it useful too, but I haven't checked with them.